index.js

```diff
@@ -11,7 +73,10 @@ module.exports = function(req) {
 		return req;
 	}
 
-	req.set('X-Csrf-Token', localStorage['XSRF.Token']);
-	req.set('X-D2L-App-Id', 'deprecated');
+	addHeaders(req);
```

Comment:
Behaviour hasn't changed in your diff so don't have to care about it, just thinking out loud.

Setting these headers right away means they can be overridden later on with `.set()`s.

If we wanted to make sure these ones were definitely being used, could switch up to
```js
var oldEnd = req.end;
req.end = function (cb) {
  addHeaders(req);
  oldEnd(cb);
};
```